internal/services/netapp/netapp_snapshot_resource.go

```diff
@@ -23,6 +23,7 @@ func resourceNetAppSnapshot() *pluginsdk.Resource {
 	return &pluginsdk.Resource{
 		Create: resourceNetAppSnapshotCreate,
 		Read:   resourceNetAppSnapshotRead,
+		// todo remove this in version 3.0 of the provider
```

Comment:
could you elaborate on what to remove and why?

Reply:
Snapshot resource in Azure has changed a while ago its type to proxied type, this does not support tags. 
The only update operation that was available on Snapshots was update on tags. So what needs to be removed is the update operation and its related function (resourceNetAppSnapshotUpdate below) since it is not doing anything.

Reply:
Snapshot resource in Azure has changed a while ago its type to proxied type, this does not support tags.
The only update operation that was available on Snapshots was update on tags. So what needs to be removed is the update operation and its related function (resourceNetAppSnapshotUpdate below) since it is not doing anything.

Reply:
oh i meant more in the comment: will update that now
```suggestion
		// todo remove this in version 3.0 of the provider as tags was the only updatable property and they can no longer be updated and will also be removed
```